docs/tutorials/eks-tutorial.md

```diff
@@ -0,0 +1,697 @@
+```
+Let's do a test deployment. Create a file called `nginx-test-deployment.yaml` with the following content:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-stock
+  labels:
+    app: nginx-stock
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-stock
+  template:
+    metadata:
+      labels:
+        app: nginx-stock
+    spec:
+      containers:
+      - name: nginx-stock
+        image: nginx
+        imagePullPolicy: IfNotPresent
+        resources:
+          #limits:
+            #memory: 768Mi
+          requests:
+            memory: 60Mi
+```
+
+And deploy it:
+```bash
+kubectl apply -f nginx-test-deployment.yaml
+```
+
+This should deploy 2 replicas of NGINX. Check the status:
+
+```bash
+kubectl get pods -o wide
+```
+Example output:
+```console
+NAME                           READY   STATUS    RESTARTS   AGE     IP                NODE                                               NOMINATED NODE   READINESS GATES
+nginx-stock-7d54d66484-k9rj5   1/1     Running   0          42s     172.16.50.197     ip-192-168-32-137.eu-central-1.compute.internal    <none>           <none>
+nginx-stock-7d54d66484-nn696   1/1     Running   0          42s     172.16.139.2      ip-192-168-103-211.eu-central-1.compute.internal   <none>           <none>
+```
+
+And let's try to check network connectivity between pods. Let's run a simple network debug container as a pod:
+
+```bash
+kubectl run tmp-shell --rm -i --tty --image nicolaka/netshoot -- /bin/bash
+```
+Expected output:
+```
+If you don't see a command prompt, try pressing enter.
+tmp-shell:~# 
+```
+
+If we issue a simple `curl` command to one of the pods IPs, we should get a response from the NGINX server:
+```bash
+tmp-shell:~# curl 172.16.139.2
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+<style>
+html { color-scheme: light dark; }
+body { width: 35em; margin: 0 auto;
+font-family: Tahoma, Verdana, Arial, sans-serif; }
+</style>
+</head>
+<body>
+<h1>Welcome to nginx!</h1>
+<p>If you see this page, the nginx web server is successfully installed and
+working. Further configuration is required.</p>
+
+<p>For online documentation and support please refer to
+<a href="http://nginx.org/">nginx.org</a>.<br/>
+Commercial support is available at
+<a href="http://nginx.com/">nginx.com</a>.</p>
+
+<p><em>Thank you for using nginx.</em></p>
+</body>
+</html>
+```
+There we go! We have a working EKS cluster, with Calico and two bare-metal nodes. Time to setup urunc! 
+
+### `urunc` setup
+
+The easiest way to setup `urunc` in such a setting is to use `urunc-deploy`. This process follows the principles of `kata-deploy` and is build to work on `k8s` and `k3s`. The process is as follows:
+
+#### 1. Clone the repo
+
+```bash
+git clone https://github.com/nubificus/urunc
+```
+> Note: For now, we use a branch. When we merge this functionality to the main branch this won't be needed.
```

Comment:
I think we can remove that. Hopefully, we will merge the `urunc-deploy` PR soon.